CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- Bump external-dns to latest release
```

Comment:
Is this change log no longer needed?

Reply:
I think this was a mistake. We don't keep track of `external-dns` on this repo? :smile:

Reply:
You're right 😄